### PR TITLE
"Retry resubmit" - allows zone.js to cleanup error stacktraces

### PR DESCRIPTION
**Pull request by aaomidi**
Reverts microsoft/azuredatastudio#7501

### CODE REVIEW

**Comment by kburtram** [comment]
Oops, I keep stepping on your PRs. 😄  I think it should be good now.

File: src/sql/setup.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-define(["require", "exports"], function (require) {
+define(["require", "exports", "vs/base/common/errors"], function (require, errors) {
```
Comment (anthonydresser) [inline src/sql/setup.js]: you are missing an import here, this line should be 
`define(["require", "exports", "vs/base/common/errors"], function (require, exports, errors) {`

Comment (aaomidi) [reply src/sql/setup.js]: -_- that makes sense.

**Comment by aaomidi** [comment]
@anthonydresser The setup file being included in the renderer file messes with the tests being run. `define` isn't defined at that point in code.

https://github.com/microsoft/azuredatastudio/pull/7502/commits/13e9d4b74b74bf3c390c72d24efcf91139478170

**Comment by aaomidi** [comment]
@kburtram I think this is ready to go in again.

**Pull request merged**